src/NosCore.GameObject/Character.cs

```diff
@@ -560,10 +560,12 @@ public void SetReputation(long reput)
         {
             Reput = reput;
             SendPacket(GenerateFd());
-            SendPacket(new MsgPacket
+            SendPacket(new SayPacket
             {
-                Type = MessageType.Whisper,
-                Message = Language.Instance.GetMessageFromKey(LanguageKey.REPUTATION_CHANGED, Session.Account.Language)
+                VisualType = VisualType.Player,
```

Comment:
this is not the way to generate a say message use the extension. Also i don't think it's a yellow one